website/views/ref.slim

```diff
@@ -1,47 +1,75 @@
-/GENERATED BY rake gen:ref
-h3
-  | lazy
-  span.constraints title='should include Rsec::Helper' helper
-pre.desc
-  | Lazy parser is constructed when parsing starts. It is useful to reference a parser not defined yet
-i.example example:
-pre.code
-  code.prettyprint.lang-rb
+/GENERATED BY rake gen:ref
+h3
+  | parse
+  span.params title='params'
+    ' (str)
+pre.desc
+  | Applied to an Rsec grammar, parses the string str. Any syntax errors are reported as INVALID_TOKEN.
+h3
+  | parse!
+  span.params title='params'
+    ' (str)
+pre.desc
+  | Applied to an Rsec grammar, parses the string str. Any synatx errors are reported in detail.
+h3
+  | r
+pre.desc
+  | Converts a terminal symbol (regex or string) into a token that can be processed by Rsec,
+    e.g. as a value assigned to a non-terminal, or a token that can have any of the following operators
+    applied to it.
+i.example example:
+pre.code
+  code.prettyprint.lang-rb
+    | utf8_tail = /[\u0080-\u00bf]/.r
+      utf8_2 = /[\u00c2-\u00df]/.r  | utf8_tail
+h3
+  | lazy
+  span.constraints title='should include Rsec::Helper' helper
+pre.desc
+  | A lazy parser for a rule is constructed when parsing starts. It is useful to reference a rule 
+    that has not been defined yet: this applies to forward references (the rule is defined later),
+    and to recursive references
+i.example example:
+pre.code
+  code.prettyprint.lang-rb
     | parser = lazy{future}
       future = 'jim'.r
+      recurse = seq( lazy{recurse} , 'a' ) | 'b'
       assert_equal 'jim', parser.parse '12323'
-h3
-  | one_of
-  span.params title='params'
-    ' (str)
-  span.constraints title='should include Rsec::Helper' helper
-pre.desc
-  | Parses one of chars in str
-i.example example:
-pre.code
-  code.prettyprint.lang-rb
+      assert_equal 'recurse', parser.parse 'baaaaa'
+h3

```

Comment:
Looks like diff functionality is getting confused: they are the same, diff is not dealing well with the interpolation.